src/@shared/Avatar.css.ts

```diff
@@ -0,0 +1,29 @@
+import { style } from "@vanilla-extract/css";
+import * as colors from "../config/colors";
+import { s } from "src/styles";
```

Comment:
```suggestion
import { s } from "src/styles";
import * as colors from "../config/colors";
```
이 파일만 import 순서가 달라서 이 부분만 수정되면 좋을 것 같습니당~

Reply:
넵넵 수정하겠습니다!